src/make.zsh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env zsh
+
+_fzf_complete_make() {
+    _fzf_complete "--ansi --tiebreak=index $fzf_options" $@ < <(grep -E '^[a-zA-Z_-]+:.*?$$' Makefile | uniq | awk -F ':' '{ print $1 }')
```

Comment:
Add `2> /dev/null` so nothing is printed when this is run outside of repository.

Reply:
fzf_options is undefined in this path. Can you just remove that?